db/seeds.rb

```diff
@@ -4,14 +4,21 @@
   {name: 'Accessories'}
   ])
 
+Brand.create!([
+  {name: 'Canon'},
+  {name: 'Sony'},
+  {name: 'Gitzo'},
+  {name: 'Lee'}
+  ])
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.